scripts/imgtool/image.py

```diff
@@ -27,10 +28,10 @@
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives import hashes
 
-IMAGE_MAGIC = 0x96f3b83d
-IMAGE_HEADER_SIZE = 32
-BIN_EXT = "bin"
-INTEL_HEX_EXT = "hex"
+IMAGE_MAGIC =         0x96f3b83d
```

Comment:
Please refrain yourself from making stylistic changes, the formatting was already PEP-8 compliant, if in doubt run flake8.

Reply:
I will revert these changes.